### ISSUE TITLE
[Suggestion] Minimize to Tray

### ISSUE THREAD

**Issue by maclanegamer** [DevilXD/TwitchDropsMiner#2]
Would be nice a minimize to tray option.

**Comment by DevilXD** [comment]
Hello o/

Interesting suggestion. The GUI library I'm using does not support tray icons, but I've found this: https://github.com/moses-palmer/pystray

Here's a quick mockup I've managed to put together using it, in about an hour or so:

![Icon](https://i.imgur.com/jolxWg0.png)
![Notification](https://i.imgur.com/Mfj0LTV.png)
![Menu](https://i.imgur.com/1vuuFVG.png)

The problem is that, adding this sparks a need for personal customization, as in - there are people who would like to minimize it the normal way (to taskbar), minimize to tray, "quit" to tray, and so on. It'd be cool to have some way of choosing this as an option, but adding any kind of options would require storing them somewhere, probably in a settings file of some sorts - but this in turn adds complexity and thus requires time. Alternatively, I could add a special separate UI button that'd minimize it to tray - this would be easy enough and avoid the need for a settings file (and all associated handling) for now.
